app/src/main/kotlin/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -342,20 +357,6 @@ class OfferActivity : AppCompatActivity(R.layout.activity_offer) {
     binding.offerScroll.layoutAnimation = animation
   }
 
-  private fun inflateMenu(loginStatus: LoginStatus) {
-    val menu = binding.offerToolbar.menu
-    menu.clear()
-    when (loginStatus) {
-      LoginStatus.Onboarding -> binding.offerToolbar.inflateMenu(R.menu.offer_menu)
-      LoginStatus.LoggedIn -> {
-        binding.offerToolbar.inflateMenu(R.menu.offer_menu_logged_in)
-        menu.getItem(0).actionView?.setOnClickListener {
-          handleMenuItem(menu[0])
-        }
-      }
-    }
-  }
```

Comment:
We had two versions of the top app bar options. One with 

- chat
- app settings
- app info
- login
- discard offer

And when already logged-in, it was just one option
- chat

This should make it show 1 menu at all times, which is the first menu minus the login option, since we are already logged in when in the offer activity.